### ISSUE TITLE
Error running xlib or pynput

### ISSUE THREAD

**Issue by Silve2611** [Nuitka/Nuitka#1623]
Cannot running xlib or pynput

Packaging works but the file cannot be run. Following error appears

```
File "/home/timebro/Git/trackingcore-linux/[core.dist/core.py](http://core.dist/core.py)", line 17, in <module>
File "/home/timebro/Git/trackingcore-linux/[core.dist/pynput/](http://core.dist/pynput/)init.py", line 40, in <module pynput>
File "/home/timebro/Git/trackingcore-linux/[core.dist/pynput/keyboard/](http://core.dist/pynput/keyboard/)init.py", line 52, in <module pynput.keyboard>
File "/home/timebro/Git/trackingcore-linux/[core.dist/pynput/keyboard/](http://core.dist/pynput/keyboard/)_xorg.py", line 34, in <module pynput.keyboard._xorg>
File "/home/timebro/Git/trackingcore-linux/[core.dist/Xlib/XK.py](http://core.dist/Xlib/XK.py)", line 66, in <module Xlib.XK>
File "/home/timebro/Git/trackingcore-linux/[core.dist/Xlib/XK.py](http://core.dist/Xlib/XK.py)", line 47, in load_keysym_group
```

* Nuitka version, full Python version, flavor, OS, etc. as output by this
  command (it does more than you think, and we are adding more all the time):

  > python -m nuitka --version
0.9

* How did you install Nuitka and Python

  pip
* The specific PyPI names and versions

  It should be taken from this output if there specific packages involved, e.g.
  `numpy`, you are expected to shorten this to the relevant ones.

  > python -m pip freeze
altgraph==0.17.2
apturl==0.5.2
blinker==1.4
Brlapi==0.8.2
certifi==2020.6.20
chardet==4.0.0
click==7.1.2
colorama==0.4.4
command-not-found==0.3
cryptography==3.3.2
cupshelpers==1.0
dbus-python==1.2.16
debugpy==1.5.1
defer==1.0.6
distro==1.5.0
distro-info==1.0
gyp==0.1
httplib2==0.18.1
idna==2.10
importlib-metadata==4.0.1
jeepney==0.7.1
keyring==23.0.1
language-selector==0.1
launchpadlib==1.10.13
lazr.restfulclient==0.14.2
lazr.uri==1.0.5
louis==3.18.0
macaroonbakery==1.3.1
more-itertools==4.2.0
netifaces==0.10.9
Nuitka==0.9
numpy==1.21.4
oauthlib==3.1.0
olefile==0.46
orderedset==2.0.3
pandas==1.3.4
pexpect==4.8.0
Pillow==8.1.2
protobuf==3.12.4
pycairo==1.16.2
pycups==2.0.1
PyGObject==3.40.1
pyinstaller==5.1
pyinstaller-hooks-contrib==2022.7
PyJWT==1.7.1
pymacaroons==0.13.0
PyNaCl==1.4.0
pynput==1.6.8
pyRFC3339==1.1
python-apt==2.2.1
python-dateutil==2.8.1
python-debian===0.1.39ubuntu1
python-xlib==0.31
pytz==2021.1
pyxdg==0.27
PyYAML==5.3.1
reportlab==3.5.66
requests==2.25.1
SecretStorage==3.3.1
simplejson==3.17.2
six==1.16.0
systemd-python==234
ubuntu-advantage-tools==27.7
ubuntu-drivers-common==0.0.0
ufw==0.36.1
unattended-upgrades==0.1
urllib3==1.26.5
wadllib==1.3.5
xdg==5
xkit==0.0.0
xlib==0.21

* Also supply a [Short, Self Contained, Correct, Example](https://sscce.org/)

```
from pynput.mouse import Listener as MouseListener
from pynput.keyboard import Listener as KeyboardListener
```

* Provide in your issue the Nuitka options used

nuitka3 --follow-imports --standalone --onefile --enable-plugin=numpy --enable-plugin=gi --assume-yes-for-downloads -o app app.py

**Comment by kayhayen** [comment]
The xlib stuff is part of the 0.9.1 hotfix I just released. The implicit dependency for pynput xorg is only on develop though, probably because it is still donein code, so this will be solved in 1.0 only.

**Comment by kayhayen** [comment]
This is part of 1.0 release. Unfortunately for macOS, I didn't yet find the time.
